src/applications/edu-benefits/10203/content/benefitSelection.jsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+
+export const benefitsLabels = {
+  chapter33: (
+    <p>
+      Post-9/11 GI Bill (Chapter 33)
+      <br />
+      <a
+        aria-label="Learn more about Post-9/11 benefits"
+        href="/education/about-gi-bill-benefits/post-9-11/"
+        target="_blank"
+      >
+        Learn more
+      </a>
+    </p>
+  ),
+  // 1995-STEM related
+  fryScholarship: (
+    <p>
+      Fry Scholarship (Chapter 33)
+      <br />
+      <a
+        aria-label="Learn more about Fry Scholarship benefits"
+        href="/education/survivor-dependent-benefits/fry-scholarship/"
+        target="_blank"
+      >
+        Learn more
+      </a>
+    </p>
+  ),
+  chapter30: (
+    <p>
+      Montgomery GI Bill (MGIB-AD, Chapter 30)
+      <br />
+      <a
+        aria-label="Learn more about Montgomery GI Bill benefits"
+        href="/education/about-gi-bill-benefits/montgomery-active-duty/"
+        target="_blank"
+      >
+        Learn more
+      </a>
+    </p>
+  ),
+  chapter1606: (
+    <p>
+      Montgomery GI Bill Selected Reserve (MGIB-SR, Chapter 1606)
+      <br />
+      <a
+        aria-label="Learn more about Montgomery GI Bill Selected Reserve benefits"
+        href="/education/about-gi-bill-benefits/montgomery-selected-reserve/"
+        target="_blank"
+      >
+        Learn more
+      </a>
+    </p>
+  ),
+  chapter32: (
+    <p>
+      Post-Vietnam Era Veterans’ Educational Assistance Program
+      <br />
+      (VEAP, Chapter 32)
+      <br />
+      <a
+        aria-label="Learn more about Post-Vietnam Era Veterans’ Educational Assistance Program benefits"
+        href="/education/other-va-education-benefits/veap/"
+        target="_blank"
+      >
+        Learn more
+      </a>
+    </p>
+  ),
+  transferOfEntitlement: (
+    <p>
+      Transfer of Entitlement Program (TOE)
+      <br />
+      <a
+        aria-label="Learn more about Transfer of Entitlement Program benefits"
+        href="/education/transfer-post-9-11-gi-bill-benefits/"
+        target="_blank"
+      >
+        Learn more
+      </a>
+    </p>
+  ),
+};
+
+// The links and labels are different from the above
+export const survivorBenefitsLabels = {
```

Comment:
this is a duplicate of https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/edu-benefits/utils/labels.jsx#L109